lib/code-ribbon-ribbon-container.js

```diff
@@ -37,6 +37,89 @@ class CodeRibbonRibbonContainer extends PaneAxis {
   }
 
   initialize() {
+    this.cr_update();
+    this.getElement().classList.add("cr-ribbon-container");
+
+    // give focus back to whichever pane had (or now has) focus
+    this.getElement().addEventListener(
+      "transitionend",
+      function(event) {
+        // crdebug("transitionend event: ", event);
+        if (
+          // ignore events from children elements bubbling up
+          event.propertyName == "transform" &&
+          event.target.classList.contains('cr-ribbon-container') &&
+          event.target.classList.contains('cr-overview-active') == false
+        ) {
+          crdebug("Refocusing activePane!");
+          atom.workspace.getActivePane().element.scrollIntoView({
+            block: "center", // vertical
+            // TODO probably want user preference for start/end/center
+            // BUG center is good for ODD n-patches, even has half patches edge
+            // inline: "nearest",
```

Comment:
Unsure what start/end/center means, could you elaborate? @robobenklein

Reply:
Has the main scroll between patches been implemented in cr-patches branch? That is what I have been using to work on this but do not see it on my environment. I remember you having it in meeting

Reply:
I tested this and the bug was not too big of an issue. Even ones did seem to be half patches but the main patch still loaded in the center of the view

Reply:
I think I will rework this entirely to always use the first patch of the view as the scroll anchor.